etc/kayobe/ansible/build-ofed.yml

```diff
@@ -0,0 +1,103 @@
+---
+- name: Build OFED packages
+  become: true
+  hosts: ofed-builder
+  gather_facts: false
+  tasks:
+    - name: Extend the home logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_home
+        size: +2G
+        resizefs: true
+
+    - name: Extend the temporary logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_var_tmp
+        size: +2G
+        resizefs: true
+
+    - name: Install package dependencies
+      ansible.builtin.dnf:
+        name:
+          - kpartx
+          - perl
+          - rpm-build
+          - automake
+          - patch
+          - kernel
+          - kernel-devel
+          - autoconf
+          - pciutils
+          - kernel-modules-extra
+          - kernel-rpm-macros
+          - lsof
+          - libtool
+          - tk
+          - gcc-gfortran
+          - tcl
+          - createrepo
+          - cmake-filesystem
+          - libnl3-devel
+          - python3-devel
+        state: latest
+        update_cache: true
+
+    - name: Set GRUB default index to saved
+      ansible.builtin.lineinfile:
+        path: /etc/default/grub
+        line: "GRUB_DEFAULT=saved"
+
+    - name: Set update default index in kernel sysconfig
+      ansible.builtin.lineinfile:
+        path: /etc/sysconfig/kernel
+        line: "UPDATEDEFAULT=yes"
+        create: true
+
+    - name: Update the default kernel entry
```

Comment:
would the reset-bls-entries.yml playbook help here?

Reply:
We can probably use this, though that playbook runs on overcloud and not the seed.

We'll also need to split build-ofed up, but that's probably a good thing as the kernel update/ofed build should probably be separate.

Reply:
This is implemented in a941098b, let me know what you think.